packages/cli/.vscode/launch.json

```diff
@@ -10,7 +10,7 @@
       "name": "Launch new command (non-interactive)",
       "skipFiles": ["<node_internals>/**"],
       "program": "${workspaceFolder}/cli.js",
-      "args": ["new", "--interactive", "false"],
+      "args": ["new", "--interactive", "false", "--app-name", "fxE2E1639996799061b9", "--capabilities", "tab", "bot", "--azure-resources", "function", "--programming-language", "typescript"] ,
```

Comment:
please verify this value is need to check in to source branch

Reply:
reverted